deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,80 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "origin.${var.r53_public_hosted_zone}"
+    origin_id   = "alb-${var.aws_region}"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
+    }
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+  http_version    = "http2"
+  comment         = "${var.project} (${var.environment})"
+
+  price_class = "${var.cloudfront_price_class}"
+  aliases     = ["${var.r53_public_hosted_zone}"]
+
+  default_cache_behavior {
+    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "alb-${var.aws_region}"
+
+    forwarded_values {
+      query_string = true
+      headers      = ["*"]
+
+      cookies {
+        forward = "all"
+      }
+    }
+
+    compress               = false
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = 0
+    default_ttl            = 0
+    max_ttl                = 300
+  }
+
+  ordered_cache_behavior {
+    path_pattern     = "static/*"
```

Comment:
>Because WhiteNoise sends appropriate cache headers with your static content, the CDN will be able to cache your files and serve them without needing to contact your application again.

I followed these instructions to setup the `ordered_cache_behavior` block:

http://whitenoise.evans.io/en/stable/django.html#restricting-cloudfront-to-static-files

Reply:
Cool. I was able to confirm that cache control headers are being set (60s).